src/pages/Lend/Filter/FilterSections/LocationMulti/FilterSectionLocationMulti.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+	<filter-menu-section title="Location">
+		<ais-refinement-list
+			:attribute="'locationFacets.lvl1'"
+			:sort-by="['name:asc']"
+			:limit="1000"
+			:transform-items="transformItems"
+		>
+			<div
+				slot-scope="{
+					items,
+					refine,
+				}"
+			>
+				<location-multi-refinements
+					:items="items"
+					:refine="refine"
+				/>
+			</div>
+		</ais-refinement-list>
+	</filter-menu-section>
+</template>
+
+<script>
+import _forEach from 'lodash/forEach';
+import FilterMenuSection from '@/pages/Lend/Filter/FilterComponents/FilterMenuSection';
+import LocationMultiRefinements from '@/pages/Lend/Filter/FilterSections/LocationMulti/LocationMultiRefinements';
+import { AisRefinementList } from 'vue-instantsearch';
+
+export default {
+	components: {
+		FilterMenuSection,
+		LocationMultiRefinements,
+		AisRefinementList,
+	},
+	methods: {
+		transformItems(items) {
+			const newItems = [];
+			const computedRegionList = {};
```

Comment:
This is a hash map that lets us easily access our regions we injected into newItems